app/controllers/contact/govuk/accessible_format_requests_controller.rb

```diff
@@ -6,7 +6,9 @@ class Contact::Govuk::AccessibleFormatRequestsController < ContactController
   before_action :show_previous_with_errors_if_missing_input
   before_action :increment_question_number_if_optional_skipped, only: [:form]
 
-  def form; end
+  def form
+    @content_base_path = content_base_path
```

Comment:
We seem to be using a mixture of class variables (`@content_base_path`) and helper methods (e.g. `question_component`). Can we make `content_base_path` a helper method for consistency, and remove the class variables?